Backend/src/constants.json

```diff
@@ -1,7 +1,8 @@
 {
-    "CAR_PORT": 4003,
-    "TEST_ADDRESS": "localhost",
-    "PI_ADDRESS": "192.168.1.15",
+    "TYP_INCOMING_DATA_PORT": 4003,
```

Comment:
TCP?

Reply:
It was supposed to be "TYP" for "TYPICAL." I just didn't want it to be too long, but I can think of something better

Reply:
NORM it is